doc/api/repl.md

```diff
@@ -256,13 +256,32 @@ function isRecoverableError(error) {
 
 By default, [`repl.REPLServer`][] instances format output using the
 [`util.inspect()`][] method before writing the output to the provided `Writable`
-stream (`process.stdout` by default). The `useColors` boolean option can be
-specified at construction to instruct the default writer to use ANSI style
-codes to colorize the output from the `util.inspect()` method.
+stream (`process.stdout` by default). The `showProxy` inspection option is set
+to true by default and the `colors` option is set to true depending on the REPLs
```

Comment:
Maybe `REPLs` -> `REPL's` (with rewrapping, unfortunatelly)?

Reply:
We use `REPLs` in one more place in the doc. Should I update that as well?

Reply:
I still am not sure. Maybe @Trott can chime in on this nit?

Reply:
Plural: `REPLs`

Possessive: `REPL's`

This is possessive, so it should get the apostrophe.